README.md

```
@@ -1,5 +1,7 @@
 ## Getting Started
 
+URL: https://singular-halva-b8695d.netlify.app/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)